driver_cpl/cime_config/config_component.xml

```diff
@@ -1849,13 +1849,13 @@
     </desc>
   </entry>
 
-  <entry id="TESTS">
-    <type>char</type>
-    <default_value>UNSET</default_value>
+  <entry id="TEST">
```

Comment:
This variable was not being used, i've repurposed it to indicate that the case is a TEST